src/components/userProfile/index.jsx

```diff
@@ -24,7 +25,7 @@ function UserProfilePage() {
                         <p> Full name </p>
                     </div>
                     <div className="infoInput">
-                        <Input value={userData.fullName} />
+                        <Input value={fullName} />
```

Comment:
```suggestion
                        <Input value={userData.name + ' ' + userData.surname} />
```
Can it be possible to rewrite like this?